customer/src/main/java/com/yas/customer/service/LocationService.java

```diff
@@ -37,15 +37,17 @@ public List<AddressDetailVm> getAddressesByIdList(List<Long> ids) {
                 .buildAndExpand()
                 .toUri();
 
-        return webClient.get()
+        return restClient.get()
                 .uri(url)
                 .headers(h->h.setBearerAuth(jwt))
                 .retrieve()
                 .onStatus(
                         HttpStatus.UNAUTHORIZED::equals,
-                        response -> response.bodyToMono(String.class).map(AccessDeniedException::new))
-                .bodyToMono(new ParameterizedTypeReference<List<AddressDetailVm>>() {})
-                .block();
+                        (request, response) -> {
```

Comment:
This exception handler should be removed, the default error handling of RestClient is fine. Apply for other files